.gitignore

```diff
@@ -1,3 +1,4 @@
+/build/
```

Comment:
Waarom moet deze er weer terug in?

Reply:
Ik krijg em in het lijstje "untracked" te zien

Reply:
Was een "left over" van het runnen van phpunit op de 2.0 branch voor alle wijzigingen (1.x code)